KSP-LogiRGB/KSP-LogiRGB/Tests/LayoutProviderTests.cs

```diff
@@ -183,7 +186,7 @@ public void TestDvorak()
                 AssertMapped(KeyCode.R, KeyCode.O);
                 AssertMapped(KeyCode.L, KeyCode.P);
                 AssertMapped(KeyCode.Slash, KeyCode.LeftBracket);
-                AssertMapped(KeyCode.Plus, KeyCode.RightBracket);
```

Comment:
While the Dvorak keycode is a plus symbol, in KSP, it is registered as a `KeyCode.Equals`